.github/workflows/load-tests.yml

```
@@ -21,7 +25,7 @@ jobs:
   setup-environment:
     timeout-minutes: 30
     runs-on: self-hosted
-    if: ${{ github.actor != 'dependabot[bot]' }}
+    if: ${{ github.actor != 'dependabot[bot]' && (contains(github.event.pull_request.labels.*.name, 'Run Load') || github.event_name == 'push')}}
```

```suggestion
    if: ${{ github.actor != 'dependabot[bot]' && (contains(github.event.pull_request.labels.*.name, 'Run Load Tests') || github.event_name == 'push')}}
```
I think it would be good to be as clear as possible with the label name here.